en-US/iot.htm

```diff
@@ -229,7 +229,7 @@ <h3 class="c-heading-4 f-lean" style="height: 56px;">Control</h3>
                     <source srcset="{{site.baseurl}}/Resources/images/Homepage/azure.png" media="(min-width:1084px)" />
                     <source srcset="{{site.baseurl}}/Resources/images/Homepage/azure.png" media="(min-width:540px)" />
                     <source srcset="{{site.baseurl}}/Resources/images/Homepage/azure.png" media="(min-width:0px)" />
-                    <img src="{{site.baseurl}}/Resources/images/Homepage/azure.png" class="mscom-image c-image" alt="Power of edege and cloud compute on IoT Core"  />
+                    <img src="{{site.baseurl}}/Resources/images/Homepage/azure.png" style="display: block; margin: 0 auto;" class="mscom-image c-image" alt="Power of edge and cloud compute on IoT Core"  />
```

Comment:
This is NO-OP. The Image is already marked as display:block and margin:none by the css classes. Adding these things again inline doesn't add anything.